packer_images/ccd_logstash.conf.in

```diff
@@ -47,6 +47,11 @@ filter{
 		target => "data_classification"
 		remove_field => ["json_data_classification"]
 	}
+	json{
+    		source => "json_supplementary_data"
+    		target => "supplementary_data"
+    		remove_field => ["json_supplementary_data"]
+    }
 	# mutate { remove_field => [ "@timestamp", "@version" ] }
 	mutate {
     	add_field => { "index_id" => "%{case_type_id}_cases" }
```

Comment:
@mario-paniccia IF this file is correct I am going to add all other jurisdictions divorcer and etc